tests/test_read_backed_experimental.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import annotations
+
+from importlib.util import find_spec
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+from scipy import sparse
+
+from anndata._core.anndata import AnnData
+from anndata.experimental import read_backed
+from anndata.tests.helpers import (
+    AccessTrackingStore,
+    as_dense_dask_array,
+    assert_equal,
+    gen_adata,
+    gen_typed_df,
+)
+
+
+@pytest.fixture(
+    params=[sparse.csr_matrix, sparse.csc_matrix, np.array, as_dense_dask_array],
+    ids=["scipy-csr", "scipy-csc", "np-array", "dask_array"],
+    scope="session",
+)
+def mtx_format(request):
+    return request.param
+
+
+@pytest.fixture(params=["zarr", "h5ad"], scope="session")
+def dskfmt(request):
+    return request.param
+
+
+@pytest.fixture(scope="session")
+def adata_remote_orig(
+    tmp_path_factory, dskfmt: str, mtx_format
+) -> tuple[AnnData, AnnData]:
+    orig_path = Path(tmp_path_factory.mktemp(f"orig.{dskfmt}"))
```

Comment:
All `tmp_path*` fixtures already return `Path` objects

```suggestion
    orig_path = tmp_path_factory.mktemp(f"orig.{dskfmt}")
```